client/components/providers/provider.js

```diff
@@ -0,0 +1,36 @@
+import React, {useEffect} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+import {fetchProvider} from '../../store/providers'
+import Container from 'react-bootstrap/Container'
+
+export default function Provider({match}) {
+  const dispatch = useDispatch()
+  const provider = useSelector(state => state.providers)
+  const user = useSelector(state => state.user)
+
+  useEffect(() => {
+    dispatch(fetchProvider(match.params.providerId))
+  }, [])
+
+  const isAuthenticatedUser = () => {
```

Comment:
we should build this into the route for this page instead of doing it on the front end. we can address this later